modules/ting_fulltext/ting_fulltext.module

```diff
@@ -47,7 +47,6 @@ function ting_fulltext_object_load($object_id) {
 
     $request = opensearch_get_request_factory()->getObjectRequest();
     $request->setObjectId($object_id);
-    $request->setProfile(variable_get('opensearch_search_profile', ''));
```

Comment:
Feeling stupid here but do you mind mentioning where this is set?

I can see it in [`opensearch_get_object()`](https://github.com/ding2/ding2/blob/039bcf756df300ba3c833a2781145f115bbf02a8/modules/opensearch/opensearch.client.inc#L65), [`opensearch_get_objects()`](https://github.com/ding2/ding2/blob/039bcf756df300ba3c833a2781145f115bbf02a8/modules/opensearch/opensearch.client.inc#L174) and [`opensearch_do_search()`](https://github.com/ding2/ding2/blob/039bcf756df300ba3c833a2781145f115bbf02a8/modules/opensearch/opensearch.client.inc#L270) but not in general.

If the current approach without a check is a problem then it seems as it [`opensearch_get_object_marcxchange`](https://github.com/ding2/ding2/blob/039bcf756df300ba3c833a2781145f115bbf02a8/modules/opensearch/opensearch.client.inc#L349) also suffers from this. It has not been updated?

Reply:
Yes, it's set in opensearch_do_search and when the new cache in opensearch gets into core it will always be set.

But for now we just testet that removing it in this situation fixed the issue